doc/contributing.rst

```diff
@@ -32,6 +32,11 @@ This dictionary contains criteria for identifying axis and coords using CF attri
 
    ~accessor.coordinate_criteria
 
+.. csv-table::
+   :file: _build/csv/coordinate_criteria.csv
```

Comment:
pre-commit is failing because this doesn't exist, it's created by `conf.py`

Reply:
I think we add `csv-table` to `ignore_directives` at the end of `setup.cfg`